system/mp/mpcomm.cpp

```diff
@@ -2165,6 +2165,11 @@ CMPConnectThread::CMPConnectThread(CMPServer *_parent, unsigned port, bool _list
     parent->mpTraceLevel = getComponentConfigSP()->getPropInt("logging/@detail", InfoMsgThreshold);
     if (listen)
     {
+        if (getComponentConfigSP()->hasProp("expert/@mpSoMaxConn"))
+            mpSoMaxConn = getComponentConfigSP()->getPropInt("expert/@mpSoMaxConn", 0);
```

Comment:
trivial: personally I'd remove the , 0 - it's the default anyway, but hasProp has established there's a value.. so slightly misleading.

Reply:
ok, I've removed the 0 arg in an amend as its such a simple change.